jinahub/rankers/DPRReaderRanker/README.md

```diff
@@ -2,8 +2,76 @@
 
  **DPRReaderRanker** performs a re-ranking of the matches for each document (question), as well as the answer spans extraction for each match. It uses the DPR Reader model to re-rank documents based on cross-attention between the question and the potential answer passages.
 
-The **DPR** model was originally proposed in [Dense Passage Retrieval for Open-Domain Question Answering](https://arxiv.org/abs/2004.04906).
+The **DPR** model was originally proposed in [Dense Passage Retrieval for Open-Domain Question Answering](https://arxiv.org/abs/2004.04906). This ranker can be used together with **[DPRTextEncoder](https://hub.jina.ai/executor/awl0jxog)** to use the entire
+DPR pipeline for QA tasks.
 
+## Usage
+
+Here's a simple example of usage for this executor
+
+```python
+from jina import Document, Flow
+from jina.types.request import Response
+
+
+question = Document(text='When was Napoleon born?')
+matches = [
+    Document(
+        text='Napoléon Bonaparte[a] (born Napoleone di Buonaparte; 15 August 1769 – 5'
+        ' May 1821), usually referred to as simply Napoleon in English'
+    ),
+    Document(
+        text='On 20 March 1811, Marie Louise gave birth to a baby boy, whom Napoleon'
+        ' made heir apparent and bestowed the title of King of Rome.'
+    ),
+]
+question.matches.extend(matches)
+
+f = Flow().add(
+    uses='jinahub+docker://DPRReaderRanker', uses_with={'num_spans_per_match': 1}
+)
+
+
+def print_matches(response: Response):
+    for match in response.data.docs[0].matches:
+        score = match.scores['relevance_score'].value
+        print(f'Napoleon was born on {match.text} [relevance {score:.2%}]')
```

Comment:
Add a comment and print `span_score` as well.

Reply:
Done